xml/schema/decoder-4-15-2.xsd

```diff
@@ -500,9 +528,15 @@
                 to an appropriate decoder definition
             </xs:documentation></xs:annotation>
           </xs:attribute>
-          <xs:attribute name="maxInputVolts" type="xs:string" />
-          <xs:attribute name="maxMotorCurrent" type="xs:string" />
-          <xs:attribute name="maxTotalCurrent" type="xs:string" />
+          <xs:attribute name="maxInputVolts" type="xs:string">
+                    <xs:annotation><xs:documentation>Deprecated, see the power element</xs:documentation></xs:annotation>
+                </xs:attribute>
+          <xs:attribute name="maxMotorCurrent" type="xs:string">
+                    <xs:annotation><xs:documentation>Deprecated, see the power element</xs:documentation></xs:annotation>
+                </xs:attribute>
+          <xs:attribute name="maxTotalCurrent" type="xs:string">
+                    <xs:annotation><xs:documentation>Deprecated, see the power element</xs:documentation></xs:annotation>
+                </xs:attribute>
```

Comment:
No need to change. But indentation doesn't seem right which makes it more difficult to read.